src/Packagist/WebBundle/Resources/views/Package/viewPackage.html.twig

```diff
@@ -10,6 +10,48 @@
     {{ parent() }}
 {% endblock %}
 
+{% block head_additions %}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.5.0/styles/monokai_sublime.min.css">
+    <script type="text/javascript">
+      WebFontConfig = {
+        google: { families: [ 'Source+Code+Pro::latin' ] }
+      };
+      (function() {
+        var wf = document.createElement('script');
+        wf.src = 'https://ajax.googleapis.com/ajax/libs/webfont/1/webfont.js';
+        wf.type = 'text/javascript';
+        wf.async = 'true';
+        var s = document.getElementsByTagName('script')[0];
+        s.parentNode.insertBefore(wf, s);
+      })();
+      function highlight_code()
+      {
+        if (typeof (Worker) === undefined) return false;
+
+        var workerFunction = new Blob(['(' + highlight_code_worker_function.toString() + ')()'], {type: "text/javascript"});
+        var localWorkerURL = URL.createObjectURL(workerFunction);
+
+        $('div.readme pre, div.readme code').each(function() {
+          var code = $(this);
+          var worker = new Worker(localWorkerURL);
```

Comment:
Good question. 

You are right, this is a 1 to 1 relation of code block to highlighting worker.
The only downside i noticed is that each worker has to `importScript` the HighlightJS resource.
This leads to multiple additional imports. I'm not sure, why this happens. Maybe my browser settings are set to always no-cache or browsers do not support that(, yet). I would expect that the resource is stored on the first load and loaded from the cache on subsequent requests.

In general, the iteration over code blocks has to go somewhere. 
You either have it on the outside using multiple workers or on the inside of one or two workers.